clustered/client/src/test/java/org/ehcache/clustered/client/replication/ActivePassiveClientIdTest.java

```diff
@@ -194,7 +197,8 @@ public void messageTrackedAndRemovedByPassiveWhenClientLeaves() throws Exception
 
     storeProxy.getAndAppend(42L, createPayload(42L));
 
-    Map<Long, EhcacheEntityResponse> responses = passiveMessageHandler.getTrackedResponsesForSegment(KEY_ENDS_UP_IN_SEGMENT_11, passiveMessageHandler.getTrackedClients().findFirst().get());
+    Map<Long, EhcacheEntityResponse> responses = activeMessageHandler.getRecordedMessages().filter(r->r.getClientSourceId().toLong() == activeMessageHandler.getTrackedClients().findFirst().get().toLong())
```

Comment:
The deprecation annotations can be removed now. (stupid github will not let me comment 'outside the lines')

Reply:
fixed